src/sql/workbench/contrib/objectExplorer/browser/serverTreeView.ts

```diff
@@ -619,6 +622,13 @@ export class ServerTreeView extends Disposable implements IServerTreeView {
 					if (errorListener) {
 						errorListener.dispose();
 					}
+
+					this._telemetryService.createActionEvent(TelemetryKeys.TelemetryView.ObjectExplorer, TelemetryKeys.TelemetryAction.ObjectExplorerFilter)
+						.withAdditionalProperties({
+							filterPropertyNames: JSON.stringify(filters.map(f => node.filterProperties.find(p => f.displayName === p.displayName).name)),
```

Comment:
Hopefully won't ever come up, but better to be safe here in case something weird happens so that telemetry stuff doesn't break the experience. 

```suggestion
							filterPropertyNames: JSON.stringify(filters.map(f => node.filterProperties.find(p => f.displayName === p.displayName)?.name)),
```